.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -21,7 +21,6 @@ Please create a second pull request at https://github.com/shopware/docs
 
 ### 5. Checklist
 
-- [ ] I have rebased my changes to remove merge conflicts
```

Comment:
why this is still valid?

Reply:
hey @shyim - I updated contribution docs, which is related to that line https://github.com/shopware/docs/pull/1620

even if we would change it to `i have resolved conflicts before creating PR` it still is a "living thing" so conflict may arise after PR is created and this checkbox becomes obsolete. In future there will be a stale bot/action to track stale and conflicted PRs so they are always up to date

Reply:
But when opening the PR the changes should still be rebased on the latest trunk, no?

Reply:
should be `synced` with the head branch, and should stay `synced` at all times during the live of PR. Does not mean that needs to be `rebased`. Even better when you see PR history to spot when branch was created and at which parts of the work head branch was merged back to it

Reply:
But a rebase is not permitted, right? I am personally in favour of seeing the clean rebased history, to see which changes were added on top of the target branch. I don't see where the mixed history and merge commits adds benefits for the reviewer. 🤔